lighthouse-core/test/audits/is-on-https-test.js

```diff
@@ -84,7 +84,7 @@ describe('Security: HTTPS audit', () => {
     // Unknown blocked resolution string is used as fallback.
     expect(result.details.items[2]).toMatchObject({
       url: 'http://localhost/image2.jpeg',
-      resolution: expect.toBeDisplayString('MixedContentBlockedLOL'),
+      resolution: 'MixedContentBlockedLOL',
```

Comment:
this is testing that it falls back to just using the protocol-provided string when it's a string that we don't recognize, so of course it's not localized.